src/esn.inbox.libs/app/controllers/resolve-emailer.controller.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+angular.module('esn.inbox.libs')
+
+  .controller('resolveEmailerController', function($scope, $q) {
+    var self = this;
+
+    self.resolveAvatar = resolveAvatar;
+
+    $scope.$watch('emailer', function(emailer) {
+      if (emailer && emailer.resolve) {
+        emailer.resolve();
+      }
+    });
+
+    /////
```

Comment:
redundant slashes ???

Reply:
No, it's a convention in our old code to separate sections. So I will just keep them